docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -1100,8 +1100,9 @@ spec:
   config: '{
             "cniVersion": "0.3.1",
             "name": "br-spoof-check",
-            "type": "cnv-bridge",
```

Comment:
Please replace cnv-bridge with bridge all over the file.

Reply:
Done